### ISSUE TITLE
Error after Obfuscation: Uncaught SyntaxError: Unexpected string

### ISSUE THREAD

**Issue by rricamar** [javascript-obfuscator/javascript-obfuscator#486]
Hi, 
After a production build with Angular 8.3, if I obfuscate my main script, I get the next error:

Chrome:

`Uncaught SyntaxError: Unexpected string :8101/main-es2015.f70397ecef4dd1524efd.js:1 `

The error message in Firefox is:

`SyntaxError: missing formal parameter`

## Your Environment

* Obfuscator version used: 0.21.0
* Node version used: 12.14.1

Also happening with Node 10.16.3 and 0.18.0

Ty for any help 🙏

**Comment by rricamar** [comment]
I can provide the original and the obfuscated script, but it is generated from the Angular Cli (20k lines more or less) ... 😱

**Comment by sanex3339** [comment]
Can you provide it?

This error happens when a function has raw value instead of parameter identifier:
```
function foo ('rawStringValue') {}
```

**Comment by rricamar** [comment]
Hi @sanex3339 , ty a lot for your quick response!
As far as I know, we don't use that type of statements. Here are the files:

[main-js-obfuscated.zip](https://github.com/javascript-obfuscator/javascript-obfuscator/files/4036084/main-js-obfuscated.zip)

The .zip contains the original file from the Angular CLI process and the obfuscated one.

Ty again 😀

**Comment by sanex3339** [comment]
Code to reproduce
```
class Foo {
    'test'() {}
}
```

With `stringArray` option it becomes
```
class _0x1aa8c8 {
    _0x2942('0x0')() {
    }
}
```

**Comment by sanex3339** [comment]
Will fixed in https://github.com/javascript-obfuscator/javascript-obfuscator/pull/489

Expect release tomorrow

**Comment by rricamar** [comment]
@sanex3339 so quick, ty a lot 🙏 

I will try new release and let you know if is working

**Comment by sanex3339** [comment]
Released

**Comment by re-jim** [comment]
This newest version 0.22.0 broke stuff for me, causing a 

`TypeError: Assignment to constant variable.` error, that doesn't happen before obfuscation, only after.

I don't have enough details at the moment to provide a full reproducible example, however. Just a heads up.

**Comment by sanex3339** [comment]
Can you send your obfuscated code and options?
This bug is critical, so i should fix it as soon as possible.

**Comment by sanex3339** [comment]
You are using `domainLock`?

**Comment by sanex3339** [comment]
Can reproduce it with enabled `disableConsoleOutput` option

**Comment by sanex3339** [comment]
Reproduced it, error when `controlFlowFlattening` option is enabled with code:
```
const test = 1;
(function () {
    console.log(1);
    console.log(2);
    console.log(3);
    console.log(4);
    console.log(5);
})();
```
 WIll fix it soon

**Comment by sanex3339** [comment]
Fixed in https://github.com/javascript-obfuscator/javascript-obfuscator/pull/491

**Comment by sanex3339** [comment]
Released as `0.22.1`, please try it

**Comment by sanex3339** [comment]
Closed. Reopen new issue if problem still exist

**Comment by rricamar** [comment]
Hi @sanex3339 , I've tried 0.22.1 and is working perfect for me 👍 

Ty for your quick response and fix!

**Comment by sanex3339** [comment]
Nice!
